content/software/zephyr-support-status/index.md

```diff
@@ -0,0 +1,144 @@
+- For ESP8684, please refer to ESP32-C2.  
+- For ESP8685, please refer to ESP32-C3.
+{{< /alert >}}
+
+## Zephyr Releases
+
+### Devices and Release Plan
+
+Espressif fully adheres to the Zephyr schedule and plans its development around Zephyr’s public roadmap. The first ready-for-production release of Zephyr for Espressif products was version 4.0, targeting the ESP32-C3.
+
+| Release Version | Date        | Expected New Device Support          | Comments | 
+|:----------------|:-----------:|:------------------------------------:|:--------:|
+| 4.2             | Jul 2025    |                                      | Current  |
+| 4.3             | Nov 2025    | ESP32-H2, ESP32-P4                   |          |
+| 4.4             | Mar 2026    | ESP32-C5, ESP32-C61, ESP32-H4        |          |
+| 4.5             | Jul 2026    |                                      |          |
+| 4.6             | Nov 2026    |                                      |          |
+| 4.7             | Mar 2027    |                                      | LTS      |
+
+
+{{< alert icon="circle-info" cardColor="#b3e0f2" iconColor="#04a5e5">}}
+- Work on devices often begins before official releases, and devices may become available in the middle of a release cycle. Therefore, the table above reflects the first official release that supports the device.
+- Dates are tentative and subject to change. The [Zephyr Release Plan](https://github.com/zephyrproject-rtos/zephyr/wiki/Release-Management#future-releases) may be updated without notice.
+{{< /alert >}}
+
+### Best Release for Espressif Devices
+
+{{< alert >}}
+**Users should always use the latest hash or commit from the [Zephyr repository](https://github.com/zephyrproject-rtos/zephyr) during the development cycle, due to ongoing updates and bug fixes.**
+{{< /alert >}}
+
+During the development stage, users are expected to use the latest available software. In other words, they should work with the most recent commit hash from the Zephyr GitHub repository, even if it does not correspond to a numbered release. The version will typically resemble *v4.0-hashnumber*.
+
+Espressif strongly recommends adhering to the rolling release model for Zephyr-based software on its devices, staying aligned with upstream developments. Espressif considers Zephyr version numbers to be weak indicators of software status. Remaining between releases is not seen as problematic, given Git’s robust tracking capabilities. Furthermore, any backfixes that diverge from upstream will result in a fork, which, according to best practices, should be avoided.
+
+Espressif follows Zephyr’s LTS releases solely for operating system bug fixes.
+
+## Disclaimers
+ 
+Espressif does not control the Zephyr Project and does not claim any ownership of it. The Zephyr Project is managed by the Linux Foundation, and Espressif participates as a regular contributor.
```

Comment:
...Linux Foundation and Espressif participates...

Reply:
A comma should be placed before "and" when it connects two independent clauses, meaning each part could stand alone as a complete sentence. However, it is not needed when "and" connects two verbs or phrases that share the same subject.